frontend/javascripts/test/model/compact_save_queue.spec.ts

```diff
@@ -0,0 +1,403 @@
+import { removeSubsequentUpdateBBoxActions } from "viewer/model/helpers/compaction/compact_save_queue";
+import type { SaveQueueEntry } from "viewer/store";
+import { describe, expect, it } from "vitest";
+
+describe("Compact Save Queue", () => {
+  it("UpdateUserBoundingBoxActions of the same tracing should be compacted", () => {
+    const actions: SaveQueueEntry[] = [
+      {
+        version: -1,
+        transactionId: "eq3scfgvoa",
+        transactionGroupCount: 1,
+        transactionGroupIndex: 0,
+        timestamp: 1747994347590,
+        authorId: "123",
+        actions: [
+          {
+            name: "updateUserBoundingBoxInVolumeTracing",
+            value: {
+              boundingBoxId: 3,
+              actionTracingId: "volumeTracing1",
+              updatedProps: {
+                boundingBox: {
+                  topLeft: [3224, 3955, 944],
+                  width: 1,
+                  height: 157,
+                  depth: 12,
+                },
+              },
+              hasUpdatedBoundingBox: true,
+              hasUpdatedName: false,
+              hasUpdatedColor: false,
+            },
+          },
```

Comment:
could you use the `updateUserBoundingBoxInVolumeTracing` action creator here (and in the rest of the file)? I think, the code will become more concise and we would also test the logic in the action creator a bit.